backend/reservations/reporting.py

```diff
@@ -139,3 +139,76 @@ def hotel_export(hotel):
 
     egest_csv(rows, fields, hotel_export_file)
     return hotel_export_file
+
+
+def rooming_list_export(hotel):
+    rooms = Room.objects.filter(name_hotel = hotel.title())
+    guests = Guest.objects.filter(name_hotel = hotel.title())
+    if rooms.count() == 0:
+        raise Exception("No rooms found for hotel %s" % hotel)
+    
+    cols = [
+        "room",
+        "room_type",
+        "first_name",
+        "last_name",
+        "secondary_name", 
+        "check_in_date",
+        "check_out_date",
+        "placed_by_roombaht",
+        "is_comp",
+        "sp_ticket_id",
+        "guest_notes",
+        "is_art",
+    ]
+    
+    rooms = Room.objects.filter(name_hotel = hotel.title())
+    if rooms.count() == 0:
+        raise Exception("No rooms found for hotel %s" % hotel)
+
+    rows = []
+    for room in rooms:
+        # hacky split to first/last name
```

Comment:
Ehhh I thought about this but some folk have multiple first names, some folk have multiple last names. I don't have a good answer for this other than to stop treating first and last names as separate fields (problem for next year).

Reply:
Yeah, for now whatever, let's just get it out. It's close enough for the spreadsheet.